qa-tests-backend/src/test/groovy/ImageScanningTest.groovy

```diff
@@ -492,6 +492,8 @@ class ImageScanningTest extends BaseSpecification {
     @Category(Integration)
     def "Verify image scan exceptions - #scanner.name() - #testAspect"() {
         Assume.assumeTrue(scanner.isTestable())
+        Assume.assumeFalse(
+            testAspect == "missing required registry" && Env.mustGetOrchestratorType() == OrchestratorTypes.OPENSHIFT)
```

Comment:
Should we have TODO here with issue about reenabling this test?

Reply:
I tried that in another PR and our linter [did not like it](https://github.com/stackrox/stackrox/pull/3797/commits/66526b2094bc48b29e9bdf78baab3a22e2b762d3) that much.
But if you insist, I think we can work that around by creating a subtask and putting a TODO with that.